tests/yam/agama/agama_auto_without_log.pm

```diff
@@ -0,0 +1,21 @@
+## Copyright 2024 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+# Summary: Wait for unattended installation to finish,
+# reboot and reach login prompt.
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+use base Yam::Agama::agama_base;
+use strict;
+use warnings;
+
+use testapi;
+
+sub run {
+    my $self = shift;
+    my $reboot_page = $testapi::distri->get_reboot();
+    $reboot_page->expect_is_shown();
+    $reboot_page->reboot();
```

Comment:
@RoyCai7 Hi Roy, all looks good. BTW, I remember you ever talked with me that logs will be added later. How about adding a comment at proper lines to remind this?

Reply:
Added, thanks for your reminder.